sdk/core/core-rest-pipeline/test/node/nodeHttpClient.spec.ts

```diff
@@ -349,6 +343,13 @@ describe("NodeHttpClient", function() {
     const promise = client.sendRequest(request);
 
     const streamResponse = new FakeResponse();
+
+    clientRequest.destroy = function(this: FakeRequest, e: Error) {
+      // give it some time to attach listeners and read from the stream
+      setTimeout(() => {
+        streamResponse.destroy(e);
+      }, 100);
```

Comment:
Does a value of 0 accomplish the same thing? Having something that's non-zero just makes me a bit nervous because that's led to flaky tests in the past.

Reply:
Yeah, 0 works. I'll go ahead and update it. The reason this is here is because Node will throw if you emit an error before any listeners are attached